src/mui/input/ReferenceArrayInput.js

```diff
@@ -278,13 +267,12 @@ function mapStateToProps(state, props) {
     };
 }
 
-const ConnectedReferenceInput = connect(mapStateToProps, {
-    crudGetMany: crudGetManyAction,
-    crudGetMatching: crudGetMatchingAction,
-})(ReferenceArrayInput);
-
-ConnectedReferenceInput.defaultProps = {
-    addField: true,
-};
+const enhance = compose(
+    connect(mapStateToProps, {
+        crudGetMany: crudGetManyAction,
+        crudGetMatching: crudGetMatchingAction,
+    }),
+    translate
+);
 
-export default ConnectedReferenceInput;
+export default enhance(ReferenceArrayInput);
```

Comment:
The exported component needs to define its default props, with `addField: true` in it